vhdbuilder/packer/pre-install-dependencies.sh

```diff
@@ -89,4 +89,19 @@ if [[ ${OS} == ${MARINER_OS_NAME} ]] && [[ "${ENABLE_CGROUPV2,,}" == "true" ]];
   enableCgroupV2forAzureLinux
 fi
 
+if [[ "${UBUNTU_RELEASE}" == "22.04" ]]; then
+  echo "Logging the currently running kernel: $(uname -r)"
+  echo "Before purging kernel, here is a list of kernels/headers installed:"; dpkg --list | grep 'linux-image\|linux-headers'
```

Comment:
I would make this use the same query that we use on line 97:

```suggestion
  echo "Before purging kernel, here is a list of kernels/headers installed:"; dpkg --list 'linux-*azure*'
```

Reply:
sounds good, changed for all